### ISSUE TITLE
test configure error, single header catch.hpp missing

### ISSUE THREAD

**Issue by Manu343726** [foonathan/type_safe#84]
It seems that the catch header is no longer hosted at `https://raw.githubusercontent.com/philsquared/Catch/master/single_include/catch.hpp`

**Comment by foonathan** [comment]
Yes, I just need to update the URL, like done with cppast already (https://github.com/foonathan/cppast/pull/62)

**Comment by Manu343726** [comment]
Alright, I'm on it

**Comment by maxyuan8** [comment]
change to https://github.com/catchorg/Catch2/tree/master/single_include/catch2/catch.hpp
